libs/wingc/src/type_check/lifts.rs

```diff
@@ -1,140 +1,93 @@
-use std::collections::{BTreeMap, BTreeSet};
+use std::collections::{BTreeMap, BTreeSet, HashMap};
 
-use itertools::Itertools;
-
-use crate::ast::Symbol;
+use crate::ast::{Symbol, UserDefinedType};
 
 use super::{ExprId, CLASS_INFLIGHT_INIT_NAME};
 
 /// A repository of lifts and captures at the class level.
 #[derive(Debug)]
 pub struct Lifts {
-	/// All the lifts. The key is "<method>/<token>"
-	lifts: BTreeMap<String, MethodLift>,
-
-	/// All the captures. The key is token.
-	captures: BTreeMap<String, Capture>,
+	// TODO: make all these private and add accessors+helper logic
+	/// All the lifts. Map from method to a map from inflight code to lift qualifications.
+	pub lifts_qualifications: BTreeMap<String, BTreeMap<String, LiftQualification>>,
```

Comment:
super nitpick / optional
```suggestion
	pub lift_qualifications: BTreeMap<String, BTreeMap<String, LiftQualification>>,
```

Reply:
I'm passing on this because we're inside a `Lifts` (plural) struct so this seems more correct.